Curtains/Curtains.Infrastructure/IRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Curtains.Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Infrastructure
+{
+    public interface IRepository<T> where T : BaseEntity
```

Comment:
What is the repository?
what does it describes?

Reply:
Default methods for our repositories